src/main/java/de/zalando/aruha/nakadi/config/ManagersConfig.java

```diff
@@ -0,0 +1,24 @@
+package de.zalando.aruha.nakadi.config;
+
+import de.zalando.aruha.nakadi.enrichment.Enrichment;
+import de.zalando.aruha.nakadi.managers.EventTypeService;
+import de.zalando.aruha.nakadi.partitioning.PartitionResolver;
+import de.zalando.aruha.nakadi.repository.EventTypeRepository;
+import de.zalando.aruha.nakadi.repository.TopicRepository;
+import de.zalando.aruha.nakadi.util.UUIDGenerator;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class ManagersConfig {
+
+    @Bean
+    public EventTypeService eventTypeManager(final EventTypeRepository eventTypeRepository,
```

Comment:
as it is Service now, I think it makes sense also to rename the method and class